CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-## v0.0.1 - 2022-01-11 - Moving
+## v0.0.2 - 2022-10-27
```

Comment:
octoDNS avoids doing releases in PRs with other changes. In this PR things can stay ??-??

```suggestion
## v0.0.2 - 2022-??-??
```